libs/net/dal/Migrations/1.0.43/Up/PreUp/00-Minister.sql

```diff
@@ -0,0 +1,5 @@
+DO $$
+BEGIN
+ALTER TABLE public."minister" ADD COLUMN "aliases" TEXT;
```

Comment:
I don't understand why you are doing this.  The DB migration should handle this.

Reply:
Forgot to remove this!